.travis.yml

```diff
@@ -84,6 +80,13 @@ jobs:
       repo: kontena/kontena
       rvm: 2.4
       env: TEST_DIR=cli
+    - stage: deploy omnibus
+      script: "rvm $TRAVIS_RUBY_VERSION do $TRAVIS_BUILD_DIR/build/travis/deploy_omnibus_macos.sh"
+      tags: true
+      os: osx
+      repo: kontena/kontena
+      rvm: 2.4
```

Comment:
2.4 or 2.5?

Reply:
I did not want to change build environment in #3294 ... that's why 2.4 is still there. This does not affect target embedded ruby version.

Reply:
BTW wouldn't it be better to bake the server/agent images and CLI releases using the same ruby version used to run the e2e tests? That would be 2.4?